src/generators/kicad.rs

```diff
@@ -123,6 +124,57 @@ impl fmt::Display for Visibility {
     }
 }
 
+impl ToLetter for ElectricKind {
+    fn to_letter(&self) -> char {
```

Comment:
Why to not implement the `fmt::Display` crait here?

Reply:
Good point! Looks like ``ToLetter`` trait isn't that useful, so I removed it completely.